### PR TITLE
Add missing visual basic sample

### LINKED ISSUE #2596: Add VB version - HttpClient class
The following topic is missing the VB sample:
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #2596
Re-create of:
https://github.com/dotnet/dotnet-api-docs/pull/2706

### CODE REVIEW

File: xml/System.Net.Http/HttpClient.xml [hunk 1]
```diff
@@ -69,8 +69,20 @@ public class GoodController : ApiController
...
+
+```vb
+  Public Class GoodController
+  Inherits ApiController  
+    ' OK  
+    Private Shared ReadOnly HttpClient As HttpClient;  
+  
+    Shared Sub New()  
+        HttpClient = New HttpClient()
+    End sub
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
    End Sub
```

File: xml/System.Net.Http/HttpClient.xml [hunk 2]
```diff
@@ -69,8 +69,20 @@ public class GoodController : ApiController
...
 }
-```  
-  
+```
+
+```vb
+  Public Class GoodController
+  Inherits ApiController  
+    ' OK  
+    Private Shared ReadOnly HttpClient As HttpClient;
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
    Private Shared ReadOnly HttpClient As HttpClient  
```

File: xml/System.Net.Http/HttpClient.xml [hunk 3]
```diff
@@ -69,8 +69,20 @@ public class GoodController : ApiController
         HttpClient = new HttpClient();
     }
 }
-```  
-  
+```
+
+```vb
+  Public Class GoodController
+  Inherits ApiController
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
    Inherits ApiController  
```

File: xml/System.Net.Http/HttpClient.xml [hunk 4]
```diff
@@ -69,8 +69,20 @@ public class GoodController : ApiController
...
     }
 }
-```  
-  
+```
+
+```vb
+  Public Class GoodController
+  Inherits ApiController  
+    ' OK
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
   
```
Though the comment appears in the C# source, it isn't helpful.

File: xml/System.Net.Http/HttpClient.xml [hunk 5]
```diff
@@ -165,9 +178,20 @@ public class GoodController : ApiController
...
     }  
 }  
-  
-```  
-  
+```
+
+```vb
+  Public Class GoodController
+  Inherits ApiController
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
    Inherits ApiController  
```

File: xml/System.Net.Http/HttpClient.xml [hunk 6]
```diff
@@ -165,9 +178,20 @@ public class GoodController : ApiController
...
 }  
-  
-```  
-  
+```
+
+```vb
+  Public Class GoodController
+  Inherits ApiController  
+    ' OK
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
   
```

File: xml/System.Net.Http/HttpClient.xml [hunk 7]
```diff
@@ -165,9 +178,20 @@ public class GoodController : ApiController
...
-  
-```  
-  
+```
+
+```vb
+  Public Class GoodController
+  Inherits ApiController  
+    ' OK  
+    Private Shared ReadOnly HttpClient As HttpClient;
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
    Private Shared ReadOnly HttpClient As HttpClient
```

File: xml/System.Net.Http/HttpClient.xml [hunk 8]
```diff
@@ -165,9 +178,20 @@ public class GoodController : ApiController
...
+
+```vb
+  Public Class GoodController
+  Inherits ApiController  
+    ' OK  
+    Private Shared ReadOnly HttpClient As HttpClient;  
+  
+    Shared Sub New()  
+        HttpClient = New HttpClient()
+    End sub
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
    End Sub
```

**Comment by Youssef1313** [comment]
@rpetrusha , Thanks.
I've applied the changes, but shouldn't the `OK` comment be removed from C# code too ?

**Comment by rpetrusha** [comment]
Optimally it should be, @Youssef1313. Would you like to include it in this PR?

**Comment by Youssef1313** [comment]
> 
> 
> Optimally it should be, @Youssef1313. Would you like to include it in this PR?

Done :)

**Comment by rpetrusha** [comment]
Thanks for removing the C# comment, @Youssef1313. I'll merge your PR now.

**Pull request merged**